__tests__/app.test.js

```diff
@@ -152,7 +152,77 @@ describe("/api", () => {
                         })
                     })
                 })
+                describe("POST", () => {
+                    test("POST /comments should return with a newly added comment", () => {
+                        return request(app)
+                        .post("/api/articles/3/comments")
+                        .send({
+                            author: "Jormanji",
+                            body: "Wow, my comment is really here!"})
+                        .expect(201)
+                        .then((response) => {
+                            expect(response.body).toHaveProperty("author", "Jormanji"),
+                            expect(response.body).toHaveProperty("body", "Wow, my comment is really here!")
+
+                            return request(app)
+                            .get("/api/articles/3/comments")
+                            .expect(200)
+                            .then(({ body: {comments}}) => {
+                                const postedComment = response.body;
+
+                                expect(postedComment).toHaveProperty("author", "Jormanji"),
+                                expect(response.body).toHaveProperty("body", "Wow, my comment is really here!")
+                            })
+                        })
+                    })
+                    test("404: sends back a 404 when the username doesn't exist", () => {
+                        return request(app)
+                        .post("/api/articles/3/comments")
+                        .send({
+                            author: "Jormanj",
+                            body: "Wow, my comment is really here!"})
+                        .expect(404)
+                        .then((response) => {
+                            expect(response.body.message).toBe("Not found")
+                        })
+                    })
+                    test("404: sends back a 404 when the article doesn't exist", () => {
+                        return request(app)
+                        .post("/api/articles/1000/comments")
+                        .send({
+                            author: "Jormanji",
+                            body: "Wow, my comment is really here!"})
+                        .expect(404)
+                        .then((response) => {
+                            expect(response.body.message).toBe("Not found")
+                        })
+                    })
+                    test("400: sends back a 400 when no body exists", () => {
+                        return request(app)
+                        .post("/api/articles/1/comments")
+                        .send({
+                            author: "Jormanji",
+                            body: ""
```

Comment:
This doesn't seem to be a bad request. If the body property was missing, then ti would really be a 400.